test/test.js

```diff
@@ -191,4 +191,18 @@ testSuite('multiple directories: handles underscored partials', () => {
   assert.is(actual, expected);
 });
 
+testSuite('handle paths with ~, test for the webpack alias', () => {
+  // enhanced-resolve needs a real file system to work
+  mock.restore();
+  const expected = path.join(process.cwd(), '/test/fixtures/foo.scss');
+  const actual = lookup({
+    dependency: '~@/foo.scss',
+    filename: './fixtures/tilde.scss',
+    directory: 'fixtures',
+    webpackConfig: path.join(__dirname, './fixtures/webpack.config.js')
```

Comment:
@Xeikim should this be `path.resolve` or `path.join`? I went with the former but since both have the same effect here I wonder what's the "right" way.